sketcherMinimizerAtom.cpp

```diff
@@ -712,7 +712,6 @@ bool sketcherMinimizerAtom::setAbsoluteStereoFromChiralityInfo()
     isRBool = !isRBool;
     isR = isRBool;
     hasStereochemistrySet = true;
-    writeStereoChemistry();
```

Comment:
It feels like, additionally, `sketcherMinimizer::writeStereoChemistry()` should clear the original bond directions, right? Otherwise, if I call `sketcherMinimizer::writeStereoChemistry()` twice, I might run into this same problem. I could imagine this happening if I minimized a molecule twice - maybe through a protocol which first did a rough minimization and then a more detailed one.

Or am I missing something fundamental? Are SketcherMolecules single-use creatures?

Reply:
not on a per-atom level. In some cases with lots of stereocenters, the only bond available to show it is shared between two stereocenters. We don't want the second one to clear the information set by the first. sketcherMinimizerMolecules are single-use creatures yes, they only live during coordinates generation, so this should be ok